cli/src/factory_ops/unboxing/network/public_mgmt

```diff
@@ -98,6 +100,8 @@ function update_cluster_pillar_for_mgmt_ips {
         provisioner pillar_set --local cluster/srvnode-1/network/mgmt_nw/public_ip_addr \"${ip_1}\"
         provisioner pillar_set --local cluster/srvnode-1/network/mgmt_nw/gateway \"${gw_ip}\"
         provisioner pillar_set --local cluster/srvnode-1/network/mgmt_nw/netmask \"${netmask}\"
+        provisioner pillar_set --local cluster/srvnode-1/network/mgmt_nw/netmask \"${search_domain}\"
```

Comment:
two things here:
- a typo `netmask`
- search domains value should be a properly constructed json list (even if it's a one item list)

Reply:
Addressed